lib/remote/jsonrpcconnection.cpp

```diff
@@ -250,9 +250,14 @@ void JsonRpcConnection::MessageHandler(const String& jsonString)
 
 bool JsonRpcConnection::ProcessMessage()
 {
+	size_t maxMessageLength = 64 * 1024;
+
+	if (m_Endpoint)
+		maxMessageLength = -1;
```

Comment:
We really shouldn't rely on the overflow behavior of signed-to-unsigned conversion. Use SIZE_MAX instead.